deepmd/nvnmd/utils/argcheck.py

```diff
@@ -0,0 +1,29 @@
+
+
+from dargs import Argument
+
+
+def nvnmd_args():
```

Comment:
You can add these arguments like `doc/train/train-input.rst`. The doc will be generated automatically in https://deepmodeling--1707.org.readthedocs.build/projects/deepmd/en/1707/train/train-input.html.

Reply:
You can either add to `doc/train/train-input.rst` or create a new file..